endpoints/openrtb2/sample-requests/custom-currency/errors/custom-rates-empty-usepbs-false.json

```diff
@@ -0,0 +1,49 @@
+{
+  "description": "usepbsrates set to false forces BidRequest to use custom currency rates but bidRequest.ext.prebid.currency.rates field is empty",
```

Comment:
They differ in line 34. This file sets `"MXN"` as the sole element in the `request.cur` array. Therefore, because PBS currency rates cannot be used, this test returns an error whereas in `endpoints/openrtb2/sample-requests/custom-currency/valid/conversion-disabled.json` we default to `"USD"`, no conversion is needed and a valid one-dollar-bid response is generated without errors.

Reply:
Ah ok! I somehow missed that. Thanks for the clarification